src/braket/experimental/autoqasm/instructions/qubits.py

```diff
@@ -25,6 +25,18 @@
 QubitIdentifierType = Union[int, oqpy._ClassicalVar, oqpy.base.OQPyExpression, str]
 
 
+def is_qubit_identifier_type(qubit: Any) -> bool:
```

Comment:
This function is to avoid `TypeError: Subscripted generics cannot be used with class and instance checks`